### PR TITLE
Identify main types file

### PR DESCRIPTION
**Pull request by rkuykendall**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "main": "dist/tester.cjs.js",
   "module": "dist/tester.esm.js",
   "browser": "dist/tester.umd.js",
+  "types": "dist/index.d.ts",
```
Comment (kbouchard) [inline package.json]: What's the advantage fo that change ?

Comment (rkuykendall) [reply package.json]: We're generating types but fields-ant ( and any other place that uses it ) can't find them.

Comment (rkuykendall) [reply package.json]: ( I think )

Comment (kbouchard) [reply package.json]: got it, thanks ! 👍

**Pull request merged**